packages/blaze/template.js

```diff
@@ -267,6 +267,8 @@ Blaze.TemplateInstance = function (view) {
    * @type {DOMNode}
```

Comment:
Is it possible to separate this into one commit which adds the feature to ddp (and doesn't touch blaze at all) followed by one commit that does the blaze/spacebars stuff?

eg, let's say we find a bug in the blaze thing and revert it.  would be a shame to lose the new feature.

if you need help splitting git commits in two i can help :)